config/crd/bases/remediation.medik8s.io_nodehealthchecks.yaml

```diff
@@ -117,6 +117,9 @@ spec:
                 - duration: 300s
                   status: "False"
                   type: Ready
+                - duration: 300s
+                  status: Unknown
```

Comment:
`"False"` is with quotes and `Unkown` without, maybe we should be consistent

Reply:
I knew this will get you.

Reply:
The reason I didn't change that is because I am afraid to trip on something stupid like False being converted to bool and not string... although the Go type should be used for reflection...
(my thoughts and fears of yaml...) - I'll change that and see what happens

Reply:
I tried editing "False" to False and the api server responded with:
```
# nodehealthchecks.remediation.medik8s.io "nodehealthcheck-sample" was not valid:                                       
# * spec.unhealthyConditions.status: Invalid value: "boolean": spec.unhealthyConditions.status in body must be of type string: "boolean"
```

So it appears the quotes are a must here.

Reply:
Cool. So maybe just add them to Unknown as well?

Reply:
Why if yaml lets me without? I prefer not to, honestly. And most of the docs example you'll find just don't use it either.

Reply:
To be consistent with the other value. That's what openshift docs do BTW.